src/buildQuery.ts

```diff
@@ -33,13 +33,19 @@ import {
 } from './types'
 
 // cache for all types
-let typeMap
+let typeMap: any
 
-export const buildQuery = (introspectionResults: Object, factory: Factory) => (
+export const buildQuery = (introspectionResults: any, factory: Factory) => (
   raFetchType: string,
-  resourceName: string,
-  params: Object
+  resName: string,
+  params: any
 ) => {
+  if (!raFetchType || !resName) return { data: null }
+
+  // We do this here because react-admin is sometimes not consistent with the case (EditGuesser, etc)
+  // eslint-disable-next-line no-param-reassign
```

Comment:
This ESLint comment isn't necessary?

Reply:
Oops, this was leftover from when I was testing this directly inside the starter code.